apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/ServerTestUtils.java

```diff
@@ -233,44 +234,112 @@ public static void createDatabaseSchema() {
-      RifFileRecords rifFileRecords = processor.produceRecords(rifFileEvent);
-      loader.process(
-          rifFileRecords,
-          error -> {
-            LOGGER.warn("Record(s) failed to load.", error);
-          },
-          result -> {
-            recordsLoaded.add(result.getRifRecordEvent().getRecord());
-          });
+    try (RifLoader loader = new RifLoader(loadAppMetrics, loadOptions); ) {
+      // Link up the pipeline and run it.
+      LOGGER.info("Loading RIF records...");
+      List<Object> recordsLoaded = new ArrayList<>();
+      for (RifFileEvent rifFileEvent : rifFilesEvent.getFileEvents()) {
+        RifFileRecords rifFileRecords = processor.produceRecords(rifFileEvent);
+        loader.process(
+            rifFileRecords,
+            error -> {
+              LOGGER.warn("Record(s) failed to load.", error);
+            },
+            result -> {
+              recordsLoaded.add(result.getRifRecordEvent().getRecord());
+            });
+      }
+      LOGGER.info("Loaded RIF records: '{}'.");
+      return recordsLoaded;
     }
-    LOGGER.info("Loaded RIF records: '{}'.");
-    return recordsLoaded;
   }
 
-  /** Calls {@link RifLoaderTestUtils#cleanDatabaseServer(LoadAppOptions)}. */
+  /** Cleans the test DB by running a bunch of <cod. */
+  @SuppressWarnings({"rawtypes", "unchecked"})
   public static void cleanDatabaseServer() {
-    RifLoaderTestUtils.cleanDatabaseServerViaDeletes(createRifLoaderOptions());
+    EntityManagerFactory entityManagerFactory = null;
+    EntityManager entityManager = null;
+    EntityTransaction transaction = null;
+    try {
+      entityManagerFactory = createEntityManagerFactory();
+      entityManager = entityManagerFactory.createEntityManager();
+
+      // Determine the entity types to delete, and the order to do so in.
+      Comparator<Class<?>> entityDeletionSorter =
+          (t1, t2) -> {
+            if (t1.equals(Beneficiary.class)) return 1;
+            if (t2.equals(Beneficiary.class)) return -1;
+            if (t1.getSimpleName().endsWith("Line")) return -1;
+            if (t2.getSimpleName().endsWith("Line")) return 1;
+            return 0;
+          };
+      List<Class<?>> entityTypesInDeletionOrder =
+          entityManagerFactory.getMetamodel().getEntities().stream()
+              .map(t -> t.getJavaType())
+              .sorted(entityDeletionSorter)
+              .collect(Collectors.toList());
+
+      LOGGER.info("Deleting all resources...");
+      transaction = entityManager.getTransaction();
+      transaction.begin();
+      for (Class<?> entityClass : entityTypesInDeletionOrder) {
+        CriteriaBuilder builder = entityManager.getCriteriaBuilder();
+        CriteriaDelete query = builder.createCriteriaDelete(entityClass);
+        query.from(entityClass);
+        entityManager.createQuery(query).executeUpdate();
+      }
+      transaction.commit();
+      LOGGER.info("Deleted all resources.");
+    } finally {
```

Comment:
I believe that sequences will not be reset? Should they be?

Reply:
👎 

But I'm lazy!

Reply:
More seriously, this is a good callout. In lieu of fixing it, I've added a comment mentioning the issue and explaining why we're not doing it right now.